lib/indocker/version.rb

```diff
@@ -1,3 +1,3 @@
 module Indocker
-  VERSION = "0.1.10"
+  VERSION = "0.1.13"
```

Comment:
I think we should not have version change in PR

Reply:
👍